test/generate_ssl_tests.pl

```diff
@@ -136,10 +136,26 @@ sub print_templates {
 sub read_config {
     my $fname = shift;
     my $provider = shift;
-    local $ssltests::fips_mode = $provider eq "fips";
+
+    my $fips_mode = $provider eq "fips";
+    local $ssltests::fips_3_4 = 0;
```

Comment:
Do we need both of these??

Reply:
Yes, we do. They are used in the .cnf.in templates.